components/reports/useQueueFilter.tsx

```diff
@@ -123,6 +123,18 @@ export const useQueueFilter = () => {
     updateFilters({ tags: [] })
   }
 
+  const setMinAccountSuspendCount = (minAccountSuspendCount?: number) => {
+    updateFilters({ minAccountSuspendCount })
+  }
+
+  const setMinReportedRecordsCount = (minReportedRecordsCount?: number) => {
+    updateFilters({ minReportedRecordsCount })
+  }
+
+  const setMinTakendownRecordsCount = (minTakendownRecordsCount?: number) => {
+    updateFilters({ minTakendownRecordsCount })
+  }
```

Comment:
Nit: It feels like exposing both `updateFilters` and a helper that wraps `updateFilters` can be confusing when using the API. Couldn't we just require users of this hook to do:

```ts
const { updateFilters } = useQuereFilter()

updateFilters({ minTakendownRecordsCount: value })
```

Reply:
agreed but we are not really using the `updateFilters` anywhere outside of the hook so I'll just remove it from the returned object.